src/terrain/clients/iplant_groups.clj

```diff
@@ -79,7 +79,11 @@
     (create-folder client user name))
   nil)
 
-;;
+;; General group functions.
+
+(defn- full-group-name [name folder]
+  (when name
+    (format "%s:%s" folder name)))
```

Comment:
The argument order kinda seems backwards here, but I wanted to be able to easily use this function with `update`.